sklearn/calibration.py

```diff
@@ -353,11 +353,10 @@ def predict_proba(self, X):
             The predicted probas.
         """
         check_is_fitted(self)
-        X = check_array(X, accept_sparse=['csc', 'csr', 'coo'],
-                        force_all_finite=False)
+
         # Compute the arithmetic mean of the predictions of the calibrated
         # classifiers
-        mean_proba = np.zeros((X.shape[0], len(self.classes_)))
+        mean_proba = np.zeros((_num_samples(X), len(self.classes_)))
```

Comment:
`predict_proba`'s docstring about `X` should be updated to reflect this. `X` does not need to be a 2-dimensional.

Reply:
Maybe we could describe `X` as 


> The samples, as accepted by `base_estimator.predict_proba` (same below)

Reply:
This comment was addressed.